### ISSUE TITLE
Hello, when I trained the refine stage, I got the white depth map. I just change the number of training data.

### ISSUE THREAD

**Issue by hahapp** [KomodoFactory/Monocular-Depth-Estimation#5]
If I just run your code, may i get the white depth map?
And, I ran the new model in your code, the loss couldn't converge.
Did you meet these situations?
Many thanks for your sharing.

**Comment by MeghaMaheshwari** [comment]
Hi @hahapp : were u finally able to get the loss to converge. I too get white depth map and wanted to check if you did something to make the network converge?
